### PR TITLE
Add privacy revision 20191021

### PR DESCRIPTION
**Pull request by kulla**
This PR updates the privacy policy on serlo.org for planned features of the KPI project (see https://github.com/serlo/p_data-analysis-and-kpis/issues/113 ).

### CODE REVIEW

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+        <p>Serlo.org nutzt personenbezogene Daten zu folgenden Zwecken:</p>
+        <ul>
+            <li>um die Dienste von Serlo anzubieten und verbessern zu können,</li>
+            <li>um zu verstehen, wie BesucherInnen Serlo.org nutzen; dadurch können wir sicherstellen, dass Serlo.org
+                ordnungsgemäß funktioniert, wir können neue Nutzungsmöglichkeiten und Features entwickeln und die Art
+                und Weise der Nutzung von Serlo.org durch unsere BesucherInnen analysieren,
+            </li>
+            <li>um Feedback zu unseren Lehrinhalten zu erhalten und diese verbessern und weiterentwickeln zu können.
+            </li>
+            <li>um die Arbeit in unserer Community zu organisieren und in Kontakt mit Autor*innen treten.</li>
```
Comment (Knorrke) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: This line was added.

Comment (Knorrke) [reply packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: Correction: in Kontakt mit Autor*innen **zu** treten

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+            registriert sind und werden anschließend gelöscht. Gesetzliche Aufbewahrungsfristen bleiben unberührt.</p>
+        <h4>Aktivitätsdaten</h4>
+        <p>Wenn du auf Serlo.org registriert und als Autor*in oder Reviewer*in aktiv bist, werden die folgenden Aktivitäten in unserer Datenbank gespeichert: Artikelerstellung, Artikeländerung, Artikelreview. Serlo erfasst jeweils deinen Nutzernamen und Zeitpunkt der Aktivität. Die Auswertung dieser Daten ermöglicht es Serlo , die Begleitung unserer Onlinecommunity zu verbessern und sicher zu stellen, dass auf jeden Beitrag eingegangen werden kann. Die Auswertung der Daten erfolgt über Monitoring- und Analyseprogrammen, welche von Serlo in der Google Cloud betrieben wird (siehe unten). Sofern du dem nicht widersprichst, kann Serlo dich kontaktieren um dich einzuladen, enger mit uns zusammenzuarbeiten.</p>
+        <p>Die Datenverarbeitung dient der Verbesserung der Wirkungsweise der Plattform. Die Datenverarbeitung geschieht auf Grundlage des Art. 6 Abs. 1 lit. f DSGVO. Der Webseitenbetreiber hat ein berechtigtes Interesse an der Auswertung der Daten zur Verbesserung des Online-Angebots.</p>
```
Comment (Knorrke) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: These lines were added

Comment (Knorrke) [reply packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: I think the text is fine. Maybe run `yarn format` here, to help with the line length?

**Comment by kulla** [comment]
Here are the current changes:

```
$ diff revision-20181201.twig revision-20191021.twig 
35a36
>             <li>um die Arbeit in unserer Community zu organisieren und in Kontakt mit Autor*innen treten.</li>
273a275,285
>         <h4>Aktivitätsdaten</h4>
>         <p>Wenn du auf Serlo.org registriert und als Autor*in oder Reviewer*in aktiv bist, werden die folgenden Aktivitäten
>         in unserer Datenbank gespeichert: Artikelerstellung, Artikeländerung, Artikelreview. Serlo erfasst jeweils deinen
>         Nutzernamen und Zeitpunkt der Aktivität. Die Auswertung dieser Daten ermöglicht es Serlo , die Begleitung unserer
>         Onlinecommunity zu verbessern und sicher zu stellen, dass auf jeden Beitrag eingegangen werden kann. Die Auswertung
>         der Daten erfolgt über Monitoring- und Analyseprogrammen, welche von Serlo in der Google Cloud betrieben wird
>         (siehe unten). Sofern du dem nicht widersprichst, kann Serlo dich kontaktieren um dich einzuladen, enger mit uns
>         zusammenzuarbeiten.</p>
>         <p>Die Datenverarbeitung dient der Verbesserung der Wirkungsweise der Plattform. Die Datenverarbeitung geschieht
>         auf Grundlage des Art. 6 Abs. 1 lit. f DSGVO. Der Webseitenbetreiber hat ein berechtigtes Interesse an der
>         Auswertung der Daten zur Verbesserung des Online-Angebots.</p>
```

These changes were already reviewed by our "Datenschutzbeauftragten".

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+            der Registrierung angegebene E-Mail-Adresse, um Dich auf diesem Wege zu informieren.</p>
+        <p>Die Verarbeitung der bei Registrierung angegebenen Daten geschieht auf Grundlage des Art. 6 Abs. 1 lit. a
+            DSGVO mit Deiner Einwilligung. Du kannst eine von Dir erteilte Einwilligung jederzeit widerrufen. Dazu
+            reicht eine formlose Mitteilung per E-Mail an uns. Die Rechtmäßigkeit der bereits erfolgten
+            Datenverarbeitung bleibt vom Widerruf unberührt.</p>
+        <p>Die bei der Registrierung erfassten Daten werden von uns gespeichert, solange Du auf unserer Website
+            registriert sind und werden anschließend gelöscht. Gesetzliche Aufbewahrungsfristen bleiben unberührt.</p>
+        <h4>Aktivitätsdaten</h4>
+        <p>Wenn du auf Serlo.org registriert und als Autor*in oder Reviewer*in aktiv bist, werden die folgenden Aktivitäten
+        in unserer Datenbank gespeichert: Artikelerstellung, Artikeländerung, Artikelreview. Serlo erfasst jeweils deinen
```
Comment (inyono) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: This also holds for other entity types, so this should be
> Inhaltserstellung, -bearbeitung, -review

Comment (inyono) [reply packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: If we add something like this, we should be generic and include the other event types too (and were they are used, e.g. activity log). Maybe a section similar to Wikipedia stating that these are all public.

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig [hunk 4]
```diff
@@ -0,0 +1,521 @@
...
+        Nutzernamen und Zeitpunkt der Aktivität. Die Auswertung dieser Daten ermöglicht es Serlo , die Begleitung unserer
+        Onlinecommunity zu verbessern und sicher zu stellen, dass auf jeden Beitrag eingegangen werden kann. Die Auswertung
+        der Daten erfolgt über Monitoring- und Analyseprogrammen, welche von Serlo in der Google Cloud betrieben wird
```
Comment (inyono) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: Doesn't matter here. We already stated that we use Google Cloud. And as long as no external entity is used for analyzing specifically that data, you don't need to repeat that everywhere.

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig [hunk 5]
```diff
@@ -0,0 +1,521 @@
...
+        (siehe unten). Sofern du dem nicht widersprichst, kann Serlo dich kontaktieren um dich einzuladen, enger mit uns
```
Comment (inyono) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig]: This section is definitely not okay. Users need to opt-in when we want to contact them outside of the stuff we already stated somewhere else in the privacy policy. So this needs to be e.g. part of the registration or asked the next time a user does one of the specified things to get permission. Also: you need to specify where/how they can opt-out again.

**Comment by inyono** [comment]
Sorry, but I don't think we can deploy that in this state. Let's talk about this again

**Comment by kulla** [comment]
https://github.com/serlo/serlo.org/pull/296 is the follow up PR -> I delete the branch for this PR...

**Pull request merged**